src/common/components/app/app.scss

```diff
@@ -48,3 +48,17 @@ footer {
     margin-bottom: var(--spacing-s) !important;
   }
 }
+
+.haitaton-pagination {
```

Comment:
For better encapsulation, could this code be in some wrapper component for Pagination rather than global style sheet?